evennia/contrib/aws-s3-cdn.py

```diff
@@ -0,0 +1,1033 @@
+"""
```

Comment:
To follow the format of other contribs, add something like this to the top:
```
AWS storage system

trhr 2020
```
You should take some credit too, man ;)